lstchain/calib/camera/r0.py

```diff
@@ -289,6 +310,22 @@ def interpolate_spikes(self, event, tel_id):
             event.r1.tel[self.tel_id].trigger_type = event.r0.tel[self.tel_id].trigger_type
             event.r1.tel[self.tel_id].trigger_time = event.r0.tel[self.tel_id].trigger_time
 
+
+    def convert_dc_to_pe(self, event, tel_id):
+        if not isinstance(event.r1.tel[tel_id].waveform, np.ndarray):
```

Comment:
What are the cases when waveform is not an ndarray? Why do you want to return nan if that happens?

Reply:
`waveform` is not an array when it has not be filled yet.
That part from the code came from LSTCameraCalibrator._calibrate_dl0() and was using CameraCalibrator._check_dl0_empty(). I could not use it as LSTR0Corrections does not herrit from CameraCalibrator, so I used the same test as in LSTR0Corrections.time_lapse_corr() and LSTR0Corrections.interpolate_spikes() instead.